src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const employeeList = [...document.querySelectorAll('tr')].slice(1, -1);
+const mainList = document.querySelector('tbody');
+const menu = document.querySelector('thead').firstElementChild.children;
+
+function salarySort(list) {
```

Comment:
It's better (and necessary with event delegation) to create one "sort" function and write different behavior depending on the column (you can pass index/column inner text, etc as argument)

Reply:
done